From 36a81359f0af5607872d20d4fdc94e9bccdd0911 Mon Sep 17 00:00:00 2001
From: Oliver Kowalke <oliver.kowalke@gmail.com>
Date: Thu, 12 Sep 2019 17:38:19 +0200
Subject: [PATCH] docu: update architectures

---
 libs/context/doc/architectures.qbk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libs/context/doc/architectures.qbk b/libs/context/doc/architectures.qbk
index 755f1479..16ea1c1c 100644
--- a/libs/context/doc/architectures.qbk
+++ b/libs/context/doc/architectures.qbk
@@ -19,6 +19,7 @@ architectures:
     [[ppc32]         [SYSV|ELF,XCOFF] [-]        [SYSV|MACH-O] [-]]
     [[ppc64]         [SYSV|ELF,XCOFF] [-]        [SYSV|MACH-O] [-]]
     [[riscv64]       [SYSV|ELF]       [-]        [SYSV]        [-]]
+    [[s390x]         [SYSV|ELF]       [-]        [-]           [-]]
     [[sparc]         [-]              [-]        [-]           [-]]
     [[x86_64]        [SYSV,X32|ELF]   [MS|PE]    [SYSV|MACH-O] [-]]
 ]
